index.html

```diff
@@ -9,7 +9,29 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
+
+    <section class="nav-open">
+      <div class="close-open">X</div>
+      <nav id="desktop-nav-open">
+        <ul class="ul-open">
```

Comment:
- [ ] As earlier pointed out by the previous code reviewer, kindly make the menu look like the figma design by having the whole content in this `ul` tag appear at the middle. You can achieve this by adding `height: 100vh` to the `section` tag and `justify-content: center` to the styling properties of this tag.